src/components/snackbar/snackbar.tsx

```diff
@@ -32,7 +32,7 @@ export class Snackbar {
      * The amount of time in milliseconds to show the snackbar. Defaults to `5000` (5 seconds).
      */
     @Prop()
-    public timeout: number;
+    public timeout?: number;
```

Comment:
or maybe it's better that I write it this way:
` public timeout?: number = 5000;`

Reply:
it will also help for a little future feature

Reply:
I think we should have a default delay (like you said 5000) but still give the opportunity to override it.

Reply:
Should we have it optional only in case the dismiss button is provided? So a snack-bar should be removed after a few seconds or by the dismiss button.

Reply:
what I think should not happen is that the timeout will override the dismiss button. for instance, if a consultant provides a dismiss button - so it means the message should be read and closed intentionally. H|owever our default 5000 will close this notification no matter the dismiss button.